app/ui-react/packages/ui/src/Layout/AppLayout.tsx

```diff
@@ -66,14 +67,28 @@ export const AppLayout: React.FunctionComponent<ILayoutBase> = ({
 
   const [breadcrumb, setHasBreadcrumb] = React.useState(null);
   const showBreadcrumb = (b: any) => setHasBreadcrumb(b);
-
+  const PageSkipToContent = (
+    <SkipToContent href="#main-content-page-layout-default-nav">
+      Skip to Content
+    </SkipToContent>
+  );
+  const [isMobileView, setIsMobileView] = React.useState(false);
+  const [isNavOpenMobile, setIsNavOpenMobile] = React.useState(false);
+  const onNavToggleMobile = () => {
+    setIsNavOpenMobile(!isNavOpenMobile);
+  };
+  const onPageResize = (props: { mobileView: boolean; windowSize: number }) => {
```

Comment:
I do hope this is somehow debounced on the PF side... if not we'll need to debounce it here.

Reply:
Yep - https://github.com/patternfly/patternfly-react/blob/master/packages/patternfly-4/react-core/src/components/Page/Page.js#L57

Reply:
Cool thanks!